src/test/java/core/basesyntax/model/fruittransaction/FruitTransactionTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.model.fruittransaction;

```

Comment:
There is no need in separate tests for models.
Their getters/setters/constructors are naturally tested when used in other test classes